kolibri/core/tasks/test/taskrunner/test_scheduler.py

```diff
@@ -20,114 +21,123 @@ def job_storage():
 
 @pytest.mark.django_db
 class TestScheduler(object):
-    def test_enqueue_at_a_function(self, job_storage):
-        job_id = job_storage.enqueue_at(local_now(), Job(id))
+    @pytest.fixture
+    def job(self):
+        return Job(id)
```

Comment:
Just curious how this works - where is `id` coming from? Is `Job()` calling a constructor with `id` as the arg?

Reply:
`id` is a built in Python function. So we're just creating a Job here with a function of `id` as its callable.